File: src/common/26.1/main/java/settingdust/preloading_tricks/v26_1/util/MinecraftAdapter.java
package settingdust.preloading_tricks.v26_1.util;

import settingdust.preloading_tricks.util.MinecraftVersion;

public class MinecraftAdapter implements settingdust.preloading_tricks.util.MinecraftAdapter {
    public MinecraftAdapter() {
        MinecraftVersion.V261.requireCurrent();
    }
}

File: src/common/1.21.1/main/kotlin/settingdust/preloading_tricks/v1_21/util/MinecraftAdapter.kt
package settingdust.preloading_tricks.v1_21.util

import net.minecraft.resources.ResourceLocation
import settingdust.preloading_tricks.util.MinecraftAdapter

class MinecraftAdapter : MinecraftAdapter {
    override fun id(namespace: String, path: String) = ResourceLocation.fromNamespaceAndPath(namespace, path)
}